src/schedlib/instrument.py

```diff
@@ -297,7 +297,7 @@ def parse_sequence_from_toast(ifile):
     """
     #columns = ["start_utc", "stop_utc", "rotation", "patch", "az_min", "az_max", "el", "pass", "sub"]
     #columns = ["start_utc", "stop_utc", "rotation", "az_min", "az_max", "el", "pass", "sub", "patch"]
-    columns = ["start_utc", "stop_utc", "rotation", "az_min", "az_max", "el", "pass", "sub", "num", "m", "patch"]
+    columns = ["start_utc", "stop_utc", "hwp_dir", "rotation", "az_min", "az_max", "el", "pass", "sub", "patch"]
 
     # count the number of lines to skip
     with open(ifile) as f:
```

Comment:
I think we can remove the `skiprows=i+2` on line 309 with the new schedules otherwise we miss the first two lines.